website/credits.php

```diff
@@ -61,6 +61,7 @@
 			<ul>
 				<li><p><strong>Alexander B.</strong> [mathfreak231] <small>&ndash; Development</small></p></li>
 				<li><p><strong>Andrew Goodsell</strong> [Zracknel] <small>&ndash; Art (battle weather backdrops)</small></p></li>
+				<li><p><strong>Annika L.</strong> [Annika] <small>&ndash; Development</small></p></li>
```

Comment:
Don't we typically skip placing usernames when the user's actual name is their username (see: me)?

Reply:
I wonder if we should stop doing that. It doesn't come up much and might be confusing to people.

Reply:
In this case it also includes the an additional initial, and `[Annika] L.` looked slightly weird?

My preference is "ask people how they would like to be credited" and that's what I've been doing.

Reply:
I think what Zarel meant was `Annika L. [Annika]`

Reply:
Yeah, I think we should start doing the redundant thing ("Annika L. [Annika]") with everyone, at this point.